FLEDGE.md

```diff
@@ -1255,6 +1257,19 @@ If the top-level auction has a `sellerCurrency` configured, this will be its cur
     *   "language-exclusions"
     *   "category-exclusions"
 
+#### 7.1.2 Downsampling
+
+This design has two main goals:
+*   Prevent sending fDO reports very often to protect user privacy:
+Only sending reports 1/1000 times the forDebuggingOnly API is called, and
+If a report is sent, “lock-out” all adtechs out of sending a report for 3 years, and
```

Comment:
```suggestion
  * If a report is sent, “lock-out” all adtechs out of sending a report for 3 years, and
```

Reply:
(Could you also replace the smart-quote characters, “ and ”, with regular quote character ", throughout?)

Reply:
Done. Thanks for noticing!